### PR TITLE
modified without moment

### PR DESCRIPTION
**Pull request by evanwmotte**

### CODE REVIEW

File: src/index.tsx [hunk 1]
```diff
@@ -250,4 +201,4 @@ TimeTable.defaultProps = {
   getDayLabel: getDefaultDayLabel,
 };
 
-export default TimeTable;
+export default TimeTable;
```
Comment (peterchappy) [inline src/index.tsx:204]: Newline EOF

File: src/stories/Timetable.stories.tsx [hunk 2]
```diff
@@ -84,7 +84,7 @@ Primary.args = {
 const HourPreview = ({ hour, defaultAttributes }: HourPreviewProps) => {
   return (
     <div {...defaultAttributes} key={hour}>
-      {hour}h
+      {hour}
```
Comment (peterchappy) [inline src/stories/Timetable.stories.tsx:87]: Should we leave the `h`?

File: src/constants.ts [hunk 3]
```diff
@@ -0,0 +1 @@
+export const DEFAULT_HOURS_INTERVAL = { from: 7, to: 24 };
```
Comment (peterchappy) [inline src/constants.ts:1]: Newline EOF

File: src/stories/Timetable.stories.tsx (hunk 2 above)
Comment (evanwmotte) [reply src/stories/Timetable.stories.tsx:87]: definitely not

File: src/types.ts [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+export interface DayColumnPreview {
+    events: Events;
+    day: string;
+    index: number;
+    rowHeight: number;
+    getDayLabel: (day: string) => string;
+    renderEvent: React.FC<EventPreview>;
+    hoursInterval: typeof DEFAULT_HOURS_INTERVAL;
+}
```
Comment (peterchappy) [inline src/types.ts:69]: EOF!!!

**Review by peterchappy — APPROVED** [review]

**Pull request merged**